### PR TITLE
Manual fix for categories enum

### PR DESCRIPTION
**Pull request by ankitd33**
Manually switching use of categories enum to str - tested out with a custom test i wrote and it does generate a link token now given a string value instead of an enum value

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,4 +1,4 @@
-# Version 2.2.9
+# Version 2.3.0
```
Comment (leewang0) [inline CHANGELOG.md:1]: We don't delete our CHANGELOG history

**Review by leewang0 — COMMENTED** [review]
Update any references to 2.2.8 or 2.2.9 in the README plz

File: CHANGELOG.md (hunk 1 above)
Comment (ankitd33) [reply CHANGELOG.md:1]: ah whoops will update

**Comment by ankitd33** [comment]
> Update any references to 2.2.8 or 2.2.9 in the README plz

i dont think there are any references to 2.2.8 or 9 in the README

**Review by leewang0 — APPROVED** [review]

**Pull request merged**